sw/device/lib/dif/dif_kmac.c

```diff
@@ -184,6 +184,10 @@ dif_result_t dif_kmac_configure(dif_kmac_t *kmac, dif_kmac_config_t config) {
                                  config.sideload);
   cfg_reg = bitfield_bit32_write(cfg_reg, KMAC_CFG_SHADOWED_ENTROPY_READY_BIT,
                                  entropy_ready);
+
+  cfg_reg = bitfield_bit32_write(cfg_reg, KMAC_CFG_SHADOWED_MSG_MASK_BIT,
+                                 config.msg_mask);
+
   mmio_region_write32(kmac->base_addr, KMAC_CFG_SHADOWED_REG_OFFSET, cfg_reg);
```

Comment:
You can use `[mmio_region_write32_shadowed]`(https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/lib/base/mmio.h;drc=dbf6803b799c214f6e75a8449216c8e556d58998;l=159) instead of calling `mmio_region_write32()` twice. (applies throughout)

Reply:
Done.